multiversx_sdk_network_providers/proxy_network_provider_test.py

```diff
@@ -1,21 +1,19 @@
-from typing import Any, Dict
-
-from multiversx_sdk_core import Address
+from multiversx_sdk_core import Address, Transaction
 
 from multiversx_sdk_network_providers.proxy_network_provider import (
     ContractQuery, ProxyNetworkProvider)
 
 
 class TestProxy:
-    proxy = ProxyNetworkProvider("https://devnet-old-gateway.multiversx.com")
+    proxy = ProxyNetworkProvider("https://devnet-gateway.multiversx.com")
```

Comment:
Not sure what's the plan for the future when it comes to testing here. I'm thinking if it might be better to have an inner http wrapper component that can be mocked and test based on that that, instead of calling public endpoint from unit tests.

Reply:
This package can be thought of the "last layer before talking to the Proxy / API", and this kind of system tests help us catch early any changes in Proxy / API responses, as well.